distribution/docs/src/main/resources/content/_running/correcting-errors.adoc

```diff
@@ -0,0 +1,10 @@
+:title: Validating Data
+:type: dataManagementIntro
+:status: published
+:summary: Methods of correcting data validation errors.
+:order: 01
+
+== {title}
+
+${branding} enables administrators to isolate metacards with data validation errors and warnings to edit the metacard to correct validation errors.
```

Comment:
✏️ ${branding} enables administrators to isolate metacards with data validation **issues and** edit the metacard to correct validation errors.